packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -19,12 +19,23 @@ import '../globals.dart' as globals;
 import '../migrations/cmake_custom_command_migration.dart';
 import 'visual_studio.dart';
 
+/// These characters appear to be fine: @%()-+_{}[]`~
```

Comment:
This is more of an aside, than official documentation, so only use two slashes (regular comment):

```suggestion
// These characters appear to be fine: @%()-+_{}[]`~
```